src/static/js/pluginfw/plugins.js

```diff
@@ -117,13 +117,14 @@ exports.getPackages = function (cb) {
           delete packages[name].parent;
         }
       
-        if (deps[name].dependencies !== undefined) flatten(deps[name].dependencies);
+        // I don't think we need recursion
+        //if (deps[name].dependencies !== undefined) flatten(deps[name].dependencies);
       });
     }
   
     var tmp = {};
     tmp[data.name] = data;
-    flatten(tmp);
+    flatten(tmp[undefined].dependencies);
```

Comment:
This worked for me with node v7 and v6. Not quite sure why it is undefined. If you have any better solution please let me know